### PR TITLE
Create glossary.md

### PR DESCRIPTION
**Pull request by rasmusei**

### CODE REVIEW

File: glossary.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+**Administrative User** – Bürokrati tagatoa kasutaja (Bürokrat backoffice user)
```
Comment (rasmusei) [inline glossary.md]:
```suggestion
**Administrative User** – Bürokratt back-office user
```

File: glossary.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
+**Administrative User** – Bürokratt back-office user
+
+**End User** – lõppkasutaja, kodanik (citizen)
+
+Bürokratt -
```
Comment (PaulaMerle) [inline glossary.md]:
```suggestion
`Bürokratt` - all automated functionalities without human interference
```

**Review by turnerrainer — CHANGES_REQUESTED** [review]
All documentation must be in English

**Review by turnerrainer — APPROVED** [review]
Keep the same styling. For instance,
`Bürokratt back-office user` (only first letter as capital) vs
`Interested 3rd Party` (all letters as capital).

In general, good enough for a start.

**Pull request merged**